src/js/components/Text/index.d.ts

```diff
@@ -9,12 +9,14 @@ import {
   PolymorphicType,
   TextAlignType 
 } from "../../utils";
+import { ExtendType } from "src/js/themes/base";
```

Comment:
This is the main thing I have a question about. In src/js/components/FormField/stories/typescript/CustomTheme.tsx, the `extend` property is used and it was complaining because `label` uses `TextProps` which did not recognize the `extend` property. So I added it. I’m not sure if that was what was desired of if, perhaps, the example in `CustomTheme` isn’t accurate.

Reply:
(NOTE: I did update this import to come from `"../../themes/base"` in a later commit.)

Reply:
I Fixed it!

Reply:
good catch!

Reply:
Thanks! 😊